atom/browser/atom_browser_client.cc

```diff
@@ -140,8 +185,7 @@ void AtomBrowserClient::OverrideSiteInstanceForNavigation(
     return;
   }
 
-  // Restart renderer process for all navigations except "javacript:" scheme.
-  if (url.SchemeIs(url::kJavaScriptScheme))
+  if (!ShouldCreateNewSiteInstance(browser_context, current_instance, url))
```

Comment:
Returning without changing `new_instance` will make Chromium use the original logic, so there is no need to duplicate the default behavior in `ShouldCreateNewSiteInstance`.